src/style.css

```diff
@@ -0,0 +1,115 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url("../src/fonts/Avenir-Book.ttf") format("ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-H";
+  src: url("../src/fonts/Avenir-Heavy.ttf") format("ttf");
+  font-weight: 300;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  --text-color: #3d4e61;
+  --white-color: #fff;
+  --border-color: #e1e7ed;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  margin: 8px;
+}
+
+.form {
+  margin: 20px auto 0;
+}
+
+.form__big-icon,
+.form__small-icon {
+  position: relative;
+}
+
+.form__big-img {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.big__input {
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  height: 70px;
+  color: var(--text-color);
+  border-radius: 4px;
+  border: 1px solid var(--border-color);
+  font-size: 16px;
+  line-height: 22px;
+  outline: none;
+  padding-left: 62px;
+  display: flex;
+}
+
+.small__input {
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  height: 42px;
+  color: var(--text-color);
+  padding-left: 33px;
+  border-radius: 4px;
+  border: 1px solid var(--border-color);
```

Comment:
don't repeat your self, you duplicate styles for inputs, create one class with common styles and reuse for two inputs